pallets/loan/src/mock.rs

```diff
@@ -224,6 +231,14 @@ impl Default for TestExternalitiesBuilder {
 	}
 }
 
+parameter_types! {
+	pub const PoolAdmin: u64 = 1;
+	pub const Borrower: u64 = 2;
+	pub const RiskAdmin: u64 = 3;
+	pub const DropInvestor: u64 = 4;
```

Comment:
We're not going to be using the DROP/TIN token names anymore for TOCC as we'll have multiple tranches. It's probably fine in the mocks/tests, but ideally we'd just call it senior/junior investors.

Reply:
true. Will change this